### DISCUSSION TITLE
BT-634-Lot in "normal" contract notices

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#1062] [Ideas]
We have noticed that the field BT-634-Lot in Contract Notices can only be properly filled in if it is a change notice, because there is a connection to BT-140.

Is there a way to make the BT-634-Lot field only available in Change Notices (in Contract Notices)?

**Comment by YvesJo** [comment]
Hi,
Allow me to rephrase your question to make sure I understand it properly.

> Currently, there are no conditionally forbidden rule that would restrict the display of the field BT-634-Lot for Contract notices. There are however some co-constraint rules that limit the options based on some criteria; in the case of contract notices the only influencing factor is the value of BT-140 which may only exist for change notices.
> It would also be good to prevent errors by hiding the BT-634 field and only have it displayed for contract notices when it is actually about a change notice.

Did I understand your request?
KR,

**Reply by JanEmde** [reply]
Yes you did. Thank you.

**Reply by YvesJo** [reply]
Thanks for the confirmation.
I will record this suggestion and expect to have this available with SDK 1.14.
KR

**Reply by JanEmde** [reply]
Thanks a lot :)
